<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<HTML>
 <HEAD>
   <TITLE> [Mono-dev] Crash when throwing exceptions after unloading	domains
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:mono-devel-list%40lists.ximian.com?Subject=%5BMono-dev%5D%20Crash%20when%20throwing%20exceptions%20after%20unloading%0A%09domains&In-Reply-To=7289B229-9554-49D2-A430-32082166B3F3%40otee.dk">
   <META NAME="robots" CONTENT="index,nofollow">
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="024073.html">
   <LINK REL="Next"  HREF="024080.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[Mono-dev] Crash when throwing exceptions after unloading	domains</H1>
    <B>Robert Jordan</B> 
    <A HREF="mailto:mono-devel-list%40lists.ximian.com?Subject=%5BMono-dev%5D%20Crash%20when%20throwing%20exceptions%20after%20unloading%0A%09domains&In-Reply-To=7289B229-9554-49D2-A430-32082166B3F3%40otee.dk"
       TITLE="[Mono-dev] Crash when throwing exceptions after unloading	domains">robertj at gmx.net
       </A><BR>
    <I>Wed Jul  4 18:31:24 EDT 2007</I>
    <P><UL>
        <LI>Previous message: <A HREF="024073.html">[Mono-dev] Crash when throwing exceptions after unloading domains
</A></li>
        <LI>Next message: <A HREF="024080.html">[Mono-dev] Crash when throwing exceptions after unloading	domains
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#24074">[ date ]</a>
              <a href="thread.html#24074">[ thread ]</a>
              <a href="subject.html#24074">[ subject ]</a>
              <a href="author.html#24074">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>Joachim Ante wrote:
&gt;<i> Hi,
</I>&gt;<i> 
</I>&gt;<i> I am getting a crash when throwing exceptions after unloading a domain.
</I>&gt;<i> 
</I>&gt;<i> I have tried debugging the issue but to no avail, but i managed to  
</I>&gt;<i> create a small sample that shows the issue.
</I>&gt;<i> <A HREF="http://bugzilla.ximian.com/show_bug.cgi?id=82007">http://bugzilla.ximian.com/show_bug.cgi?id=82007</A>
</I>&gt;<i> 
</I>&gt;<i> There are a couple of weird things about it, for example if i run the  
</I>&gt;<i> code that throws / catches the exception before unloading the domain  
</I>&gt;<i> one time. And one time after unloading the domain, the crash goes away.
</I>
Instinctively I'd say it crashes because you're calling mono_jit_exec ()
twice. Try mono_runtime_exec_main () instead.

Robert


</PRE>

<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="024073.html">[Mono-dev] Crash when throwing exceptions after unloading domains
</A></li>
	<LI>Next message: <A HREF="024080.html">[Mono-dev] Crash when throwing exceptions after unloading	domains
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#24074">[ date ]</a>
              <a href="thread.html#24074">[ thread ]</a>
              <a href="subject.html#24074">[ subject ]</a>
              <a href="author.html#24074">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="http://lists.ximian.com/mailman/listinfo/mono-devel-list">More information about the Mono-devel-list
mailing list</a><br>
</body></html>
